assets/js/vue/img.js

```diff
@@ -17,60 +24,94 @@ export default {
       ];
     },
     source() {
-      if (Tools.isTrue(this.cloudinary)) {
-        return `${basePath}${this.urlParams}${this.img}`;
-      } else {
-        return `${this.img}`;
-      }
+      return this.fallback || this.getCloudinaryLink(this.img);
     },
     loading() {
       return this.lazy ? 'lazy' : null;
     },
-    srcset() {
-      if (!this.cloudinary) return;
-
-      return;
-    },
     sizes() {
-      if (!this.cloudinary) return;
-
-      const Preset = this.getPreset();
-
-      return Preset.sizes;
-    },
-    width() {
-      // TODO implement width calculation
-    },
-    height() {
-      // TODO implement height calculation
-    },
-    urlParams() {
-      return this.getUrlParams();
+      return this.preset?.sizes;
```

Comment:
Das funktioniert so nicht. Glaube im Jekyll Kontext ist this.preset zwar ein String, aber übergeben wird das dahinterliegende Objekt. Das muss auf das Preset in setPreset referenzieren. Sizes werden nämlich keine Ausgegeben.